part-2.py

```diff
@@ -4,13 +4,38 @@
 # the appropriate comment.
 
 # search
-
-
+def search(array,query):
+    if array == []:
+        return False
+    if array[0] == query:
+        return True
+    x = array[0]
```

Comment:
Minor note, it looks like x is not used. 

Another way to achieve this slicing is with `array[1:]`